packages/order-utils/src/order_validation_utils.ts

```diff
@@ -225,6 +308,44 @@ export class OrderValidationUtils {
             zrxAssetData,
         );
     }
+    /**
+     * Validate a call to FillOrder and throw if it wouldn't succeed. This
+     * calls validateFillOrderThrowIfInvalidAsync under the hood
+     * @param erc20ProxyAddress A valid address where an ERC20ProxyContract is deployed
+     * @param erc721ProxyAddress A valid address where an ERC721ProxyContract is deployed
+     * @param zrxTokenAddress A valid address where ZRXTokenContract is deployed
+     * @param provider The same provider used to interact with contracts
+     * @param signedOrder SignedOrder of interest
+     * @param fillTakerAssetAmount Amount we'd like to fill the order for
+     * @param takerAddress The taker of the order
+     */
+    public async simpleValidateFillOrderThrowIfInvalidAsync(
```

Comment:
Wasn't this being replaced with the `callAsync` to `fillOrder`? Don't think we need this.

Reply:
Removed